src/components/Confirm/CaptureViewer/hocs.js

```diff
@@ -3,7 +3,7 @@ import { blobToLossyBase64 } from '~utils/blob'
 import { createObjectURL, revokeObjectURL } from '~utils/objectUrl'
 
 export const withBlobPreviewUrl = (WrappedComponent) =>
-  class extends Component {
+  class BlobPreviewUrlComponent extends Component {
```

Comment:
This makes wrapped components much more readable, although it would be good to use less HOCs going forward (out of scope).

Reply:
Yes this is exactly what I want to raise: avoid using HoC and moving to [Hooks](https://reactjs.org/docs/hooks-intro.html#its-hard-to-reuse-stateful-logic-between-components) seem to be a promising solution here.